src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Successfully fetched data from the database. Params: userId = {}", userId);
```

Comment:
it is rather a debug log

also you need to think of the log content - I would rather log smth like `Successfully fetched data for user: {}`